test/testdata/lsp/code_actions/delete_unsafe.rb

```diff
@@ -0,0 +1,15 @@
+# typed: true
```

Comment:
Can you add a test case where the action is called on something other than `T.unsafe`? Ex.
```
0.even?
#  ^ apply-code-action: [A] Delete T.unsafe
```

Reply:
Given the way that our assert-no-code-action mechanism works, I don't think this is a worthwhile test to add. I'd have to make a whole empty file and assert that there are no code actions in that file with the kind "refactor.rewrite".

But it could easily be the sort of thing in the future where we add more refactor.rewrite code actions that apply in that file, and it wouldn't be useful to have the test fail in that case.

Also, I don't even know how those assert-no-code-action tests work? Because it would have to like simulate a code action at all points, but I'm sure it doesn't do that.

Anyways, I don't think this is a useful test to add given our test harness. If you disagree feel free to push back.